scripts/mint.ts

```diff
@@ -109,6 +109,50 @@ async function mintFor(tokenId: string, recipient: string) {
   );
 }
 
+async function mintForRange(tokenIdRange: string, recipient: string) {
+  const starsRecipient = toStars(recipient);
+
+  // Parse string from "1,10" -> "1" and "10"
+  let split = tokenIdRange.split(",");
+  let start = parseInt(split[0]), end = parseInt(split[1]);
+
+  // Verify proper range
+  if (isNaN(start) || start < 1) throw new Error("Invalid start ID");
+  if (isNaN(end) || end < 1) throw new Error("Invalid end ID");
+  if (start > end) throw new Error("Invalid range");
+
+  console.log('Minting tokens', start + '-' + end, 'for', starsRecipient);
+
+  // Loop through range and generate contract messages.
+  let msgArray = new Array();
+  for (let i = start; i <= end; i++) {
+    let msg = {
+      mint_for: { token_id: i, recipient: starsRecipient },
+    };
+    let executeContractMsg: MsgExecuteContractEncodeObject = {
+      typeUrl: '/cosmwasm.wasm.v1.MsgExecuteContract',
+      value: MsgExecuteContract.fromPartial({
+        sender: config.account,
+        contract: config.minter,
+        msg: toUtf8(JSON.stringify(msg)),
+        funds: [...[]],
+      }),
+    };
+    msgArray.push(executeContractMsg);
+  }
+
+  const client = await getClient();
+
+  // Execute all messages.
+  const result = await client.signAndBroadcast(
+    config.account,
+    msgArray,
+    'auto',
```

Comment:
for a large enough msgArray this may exceed gas limits. if that's the case, it'll just throw an execution error and they will need to narrow the range. beyond the scope of this PR, but just something to be aware of.